packages/builder/src/builtin-tasks/deploy.ts

```diff
@@ -35,8 +32,18 @@ export function loadFilenames (directory: string): string[] {
   return files.sort(cmpStr);
 }
 
+function clearCheckpointFiles (scriptNames: string[]): void {
+  for (const scriptName of scriptNames) {
+    try {
+      fs.unlinkSync(toCheckpointFileName(scriptName));
+    } catch (e) {
+      // ignored
+    }
+  }
+}
+
 async function doDeploy ({ fileNames, force }: TaskArgs, runtimeEnv: AlgobRuntimeEnv): Promise<void> {
-  const log = debug("builder:core:tasks:deploy");
```

Comment:
let's have something shorter. Not sure why we need `core` and `tasks`. Also, the tool name is `algob`.
Let's use: `algob:deploy`

Reply:
It was left from previous project. Then I should update all tags.

Reply:
was this `builder:core`?

Reply:
The name is constructed this way:
`builder:core` for the package and `tasks:deploy` for deploy task. Which makes sense.

I can change it to `algob:core:...` but it doesn't make much difference since we don't use command debugging for now.

Reply:
changed to `algob:task:deploy` and run too